Game/RainMeadow.GameplayHooks.cs

```diff
@@ -15,6 +15,36 @@ public void GameplayHooks()
             On.Creature.Grasp.ctor += GraspOnctor;
             On.PhysicalObject.Grabbed += PhysicalObjectOnGrabbed;
             On.Creature.SuckedIntoShortCut += CreatureSuckedIntoShortCut;
+            On.RegionGate.Update += RegionGateTransition;
+        }
+
+        private void RegionGateTransition(On.RegionGate.orig_Update orig,  RegionGate self, bool eu)
```

Comment:
Throw this into RainMeadow/Story/RainMeadow.StoryHooks.cs

I've been keeping Story only gamehooks here